mantle/platform/cluster.go

```diff
@@ -111,10 +117,39 @@ func (bc *BaseCluster) Machines() []Machine {
 	return machs
 }
 
+func (bc *BaseCluster) appendSSH(m Machine) error {
+	sshConfig, err := os.OpenFile(filepath.Join(bc.rconf.OutputDir, "ssh-config"), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		return errors.Wrapf(err, "creating ssh config")
+	}
+	defer sshConfig.Close()
+	sshBuf := bufio.NewWriter(sshConfig)
+
+	machIP := m.IP()
+	idx := strings.LastIndex(machIP, ":")
+	if idx == -1 {
+		panic(fmt.Sprintf("Malformed machine IP %s", machIP))
+	}
+	_, err = fmt.Fprintf(sshBuf, `Host %s
```

Comment:
Can we do something like: if it's the first machine in the cluster, we also alias it to `vm` or something? That way in the common case of just single node clusters, we don't have to fish around for the UUID, we can just `ssh -F path/to/ssh-config vm`.

Reply:
I was thinking for now we just hack things by doing:
`sed -ie s,/^Host.*,Host vmcheck,`
Doing what you're asking makes total sense, but the current layering in the code makes this a bit awkward.

Well hmm, I guess we could base the name on the size of the machine hashmap.

Reply:
> Well hmm, I guess we could base the name on the size of the machine hashmap.

Yup, makes sense to me. Anyway, no need to block on this. We can revisit later.